chapter7/examples/Flexbox/styling.css

```diff
@@ -1,44 +1,48 @@
-/*make sure that dimensions of any box includes its border and padding, giving easy way to size elements*/
+/* make sure that dimensions of any box includes its border and padding, giving easy way to size elements */
 * {
     box-sizing: border-box;
 }
 
-.container{
-    background-color: #07575B;
+.container {
+    background-color: #07575b;
     height: 200px;
     margin: 10px;
     display: flex;
     flex-wrap: nowrap /*wrap, nowrap, wrap-reverse*/;
-    flex-direction: row /*column-reverse, row, row-reverse, column, column-reverse*/;
-    justify-content: center /*center, flex-start, flex-end, space-around, space-between*/;
+    flex-direction: row
+        /*column-reverse, row, row-reverse, column, column-reverse*/;
+    justify-content: center
+        /*center, flex-start, flex-end, space-around, space-between*/;
     align-items: stretch /*center, flex-start, flex-end, stretch, baseline*/;
-    align-content: center /*center, stretch, flex-start, flex-end, space-between, space- around*/;
+    align-content: center
+        /*center, stretch, flex-start, flex-end, space-between, space- around*/;
 }
 
-.container > div{
-    background-color: #66A5AD;
+.container > div {
+    background-color: #66a5ad;
     color: aliceblue;
     text-align: center;
     padding: 20px;
     margin: 1px;
-    border: 3px #C4DFE6 solid;
-/*
+    border: 3px #c4dfe6 solid;
+
+    /*
     flex-grow: 1;
     flex-shrink: 1;
     flex-basis: 150px;
 */
-/*
+
+    /*
     flex: initial;
     flex: auto;
     flex: none;
 */
```

Comment:
Is this supposed to be empty? @JJ27

Reply:
Ah I think I missed that, it should be all good now! @phrdang